doc/running/profiling.rst

```diff
@@ -12,3 +16,15 @@ In addition to using this array context, you also need to enable profiling in th
 Note that profiling has a performance impact (~20% at the time of this writing).
 
 .. automodule:: mirgecom.profiling
+
+
+Time series logging
+-------------------
+
+Mirgecom supports logging of simulation and profiling quantities with the help
+of :mod:`logpyle`. Logpyle requires
+classes to describe how quantities for logging are calculated. For mirgecom, these classes are described below.
+
+.. automodule:: mirgecom.logging_quantities
+
+An overview of how to use logpyle is given `here <https://logpyle.readthedocs.io/en/latest/log.html#how-do-i-use-this>`_.
```

Comment:
FYI: You can link to sections in other docs using Intersphinx, too. https://docs.readthedocs.io/en/stable/guides/intersphinx.html

Reply:
Done